### ISSUE TITLE
opt-13b checkpoint missing final_layer_norm weights in pretrained checkpoint

### ISSUE THREAD

**Issue by zw123han** [huggingface/transformers#20930]
### System Info

Hello. Apologies if this has been brought up before, but it seems at least the flax version of opt-13b when using from_pretrained is missing weights for:

- model/decoder/final_layer_norm/bias
- model/decoder/final_layer_norm/scale

Every other version of the model I've tested didn't have this issue. You can see the message below.
![image](https://user-images.githubusercontent.com/67124639/209904124-1aed006f-d20b-4cd8-9602-fe90f8da5d71.png)

### Who can help?

@ArthurZucker 

### Information

- [X] The official example scripts
- [X] My own modified scripts

### Tasks

- [ ] An officially supported task in the `examples` folder (such as GLUE/SQuAD, ...)
- [ ] My own task or dataset (give details below)

### Reproduction

I'm using a modified version of `run_clm_flax.py` under `examples/flax/language-modelling`, although my modification below should not make an impact and the default script should have the same issue. 

`config = AutoConfig.from_pretrained(
            'facebook/opt-13b',
        )
`

`model, params = FlaxAutoModelForCausalLM.from_pretrained(
            'facebook/opt-13b',
            config=config,
            seed=42,
            _do_init = False, 
        )`

`params = model.init_weights(model.key, model.input_shape, params).unfreeze()` # inside a jit function
        

### Expected behavior

You will notice the message above which shows missing weights for only the last layer of this specific model.

**Comment by zw123han** [comment]
Hi Arthur, thanks for the help! Managed to convert PyTorch weights and all of them are there!

Although, for anyone else using `_do_init = False` in `from_pretrained`, I had to convert and save to Flax weights first to a local dir as the `from_pt = True` flag will attempt to internally call `model.params`.
